confmap/provider/eecprovider/README.md

```diff
@@ -0,0 +1,28 @@
+# EEC Provider
+
+This is an OpenTelemetry Collector `confmap.Provider` module that allows the
+Collector to be configured with the Dynatrace Extensions Execution Controller.
+
+> [!WARNING]
+> This is not intended for direct customer use, but is only intended for use by
+> Collectors installed and managed by Dynatrace. Configuring this
+> component directly is not supported.
+
+## Configuration
+
+This confmap Provider can be configured by pasing query parameter-formatted values
+inside the fragment of the URL given to the config flag. For example:
+
+```text
+dynatrace-otel-collector --config=eec://my.eec.host:31098#refresh-interval=5s&auth-header=Api-Token&auth-file=/var/private/token.key
+```
+
+### Options
+
+| Key | Default | Description |
+|-----|---------|-------------|
+| auth-env |  None | An environment variable that should be read to get a plaintext API token or other key to be used in an HTTP header. Mutually exclusive from `auth-file`, passing both options should result in an error. |
```

Comment:
"should result in an error" or will result in an error? 🤔

Reply:
Good catch. "Will result in an error" is right.